### ISSUE TITLE
A voter should only be allowed to cast one vote in a proposal.

### ISSUE THREAD

**Issue by Phyloiota** [soonlabs/soonaverse-dao#198]
Currently, it seems I am allowed to vote unlimited times in a proposal.
You need to implement the limitation to only one vote per account, and maybe an optional possibility to change a vote one time (which cancels the vote done before).
If there would be things like a required minimum threshold required in a proposal I can easily reach it by issuing those votes myself it seems.
Here I voted 2 times no and 1 time yes and I can still continue to vote on the proposal. I don't know if you count these additional votes, but it's definitely confusing. So if someone attempts a second vote in a proposal in which he already voted you should give a message saying "you have already voted on this proposal" and not allow the transaction.
<img width="286" alt="image" src="https://user-images.githubusercontent.com/77154511/146281280-a2909164-c348-416b-ac7a-566150223cee.png">

**Comment by adamkundrat** [comment]
Thank you for reporting.

We used to allow change of an opinion but it was confusing. We've tweaked UI a lot in the upcoming release. 

Change of an opinion is still supported but disabled within the front-end for now to make it easier to understand. 

Feel free to check it out at edge.soonaverse.com. We will release this by Friday.

https://user-images.githubusercontent.com/130610/146309112-401c3934-deaf-4cbd-8214-fd5057b2ff99.mov
